package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cronstrue",
   "title": "cRonstrue",
-  "version": "1.69.0",
+  "version": "1.70.0",
```

Comment:
No need to bump version - the release script [does that automatically](https://github.com/bradymholt/cRonstrue/blob/master/.travis.yml#L7).

Reply:
Reverted

Reply:
I'm not seeing the revert on this.